plugins/sonar-xoo-plugin/src/test/java/org/sonar/xoo/coverage/ItCoverageSensorTest.java

```diff
@@ -60,7 +61,7 @@ public void testDescriptor() {
 
   @Test
   public void testNoExecutionIfNoCoverageFile() {
-    DefaultInputFile inputFile = new DefaultInputFile("foo", "src/foo.xoo").setLanguage("xoo");
+    InputFile inputFile = new TestInputFileBuilder("foo", "src/foo.xoo").setLanguage("xoo").build();
```

Comment:
And this decouples InputFile from DefaultInputFile in the tests. Note that it involves IndexedFile too.